src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -143,35 +143,37 @@ struct npc_doomfire_spirit : public ScriptedAI
 {
     npc_doomfire_spirit(Creature* creature) : ScriptedAI(creature){ }
 
+    float const m_turnConstant = 0.785402f;
```

Comment:
Please don't use hungarian notations (the m_* thing). I know its common on MaNGOS but tc/ac has moved past it quite a while ago

Reply:
Adjusted, please let me know if it's still not up to snuff.